activities/index.md

```diff
@@ -4,16 +4,19 @@ type: Index
 
 # Activities
 
-Activities of the Foundation for Public Code that directly serve to further the [mission](../organization/mission.md):
+Higher level activities of the Foundation for Public Code that directly serve to further the [mission](../organization/mission.md):
 
 * [Codebase stewardship](codebase-stewardship/index.md): Enabling a codebase to attain the sustainability, marketability, support and quality needed to be successfully reused and community developed.
-* [Codebase auditing](codebase-auditing/index.md): Performing standards compliance checking on codebases, including on the Standard for Public Code and project specific standards.
 
 Activities that support the above as well as make staff operations work:
 
+* [Codebase auditing](codebase-auditing/index.md)
+* [Codebase stewardship assesment](codebase-stewardship-assesment/index.md)
```

Comment:
```suggestion
* [Codebase stewardship assessment](codebase-stewardship-assessment/index.md)
```

Reply:
The folder need to change name as well, not only the link to it.

Reply:
I've renamed everything. Largest Fix typo commit ever: https://github.com/publiccodenet/about/pull/649/commits/75fa3101e12d0f2efdb62562aff5b0218be0ac36